packages/toolkit/form-field/form-fieldset.css

```
@@ -1,7 +1,79 @@
+  --_fieldset-clr-info: #007bc7;
+  --_fieldset-clr-success: #3f8f4f;
+  --_fieldset-clr-danger: #db1818;
+  --_fieldset-clr-warning: #a16207;
+  --_fieldset-clr-warning-strong: #92400e;
+
+  --_fieldset-bg-danger-soft: #fdebeb;
+  --_fieldset-bg-selection-danger: #fbdddd;
+  --_fieldset-bg-warning-soft: #fff6e8;
+  --_fieldset-border-danger-soft: rgb(219 24 24 / 0.18);
+  --_fieldset-border-warning-soft: rgb(161 98 7 / 0.2);
+  --_fieldset-message-animation-default: ngxStatusSlideIn 300ms
```

Stylelint (17.7.0)

[error] Expected "ngxStatusSlideIn" to be "ngxstatusslidein" (value-keyword-case)

(value-keyword-case)